<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QComboBox" name="device_name_list">
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="resolution">
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="fps">
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_camera">
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">camera_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_camera">
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">camera_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVideoDisplay">
    <property name="title">
     <string>Operation</string>
    </property>
   </widget>
   <addaction name="menuVideoDisplay"/>
  </widget>
  <action name="actionStart_Camera">
   <property name="text">
    <string>Start Camera</string>
   </property>
  </action>
  <action name="actionStop_Camera">
   <property name="text">
    <string>Stop Camera</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>start_camera</sender>
   <signal>toggled(bool)</signal>
   <receiver>device_name_list</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_camera</sender>
   <signal>toggled(bool)</signal>
   <receiver>resolution</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_camera</sender>
   <signal>toggled(bool)</signal>
   <receiver>fps</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="camera_group"/>
 </buttongroups>
</ui>
